src/components/code-snippet-title/code-snippet-title.js

```diff
@@ -28,12 +28,12 @@ export default class CodeSnippetTitle extends React.PureComponent {
   render() {
     const { link, toggle } = this.props;
     return (
-      <div className="flex-parent-mm flex-parent--space-between-main-mm mb6">
-        <div className="flex-child">
+      <div className="flex-mm flex--space-between-main-mm mb6">
+        <div>
```

Comment:
I noticed an issue that I think it present in the current implementation: on smaller screens we want the file name and GitHub link to stack.

I think we could make that happen by adding `flex-mm` here and then removing `inline-block` from line 8 (may want to remove `mb6` from line 8 as well) and `inline-block` from line 16.

```suggestion
        <div className="flex-mm mb6">
```

Or we could update `ml12-mm` on line 16 to `ml12` to prevent the loss of space between the filename and GitHub link on smaller screens.

Reply:
![image](https://user-images.githubusercontent.com/19801577/133507103-ec8226ba-1d87-4b4c-b55f-dbdedcae63d3.png)

@katydecorah thank you for the suggestions. Let me know what you think! I kept `mb6` since it looks better when the filename and link are stacked to have some space in between them.